File: C files/compilation-speed/io.c
#include <stdio.h>

int main() {

    FILE *file = fopen("test.txt", "w");

    for (int i = 0; i < 1000; i++) {
        fprintf(file, "Line %d\n", i);
    }

    fclose(file);

    return 348333;
}

File: Go Files/compilation-speed/io.go
package main

import (
	"fmt"
	"os"
)

func main() { 

    file, err := os.Create("test1.txt") 

    defer file.Close()

    for i := 0; i < 1000; i++ {
        fmt.Fprintf(file, "Line %d\n", i)
    }
}
